js/app/demo/rulesConfig.jsx

```diff
@@ -50,7 +46,6 @@ const RulesConfig = React.createClass({
             change[key][0] = e.target.checked ? 'error' : 'off';
         }
         this.setState(change);
-//      this.props.onUpdate(this.state)
```

Comment:
Ultimately this is what's slowing us down, the rest of the stuff I think had little effect (still testing). I'm going to try running this async or a few other things...